readme.md

```diff
@@ -246,6 +246,7 @@ If you want to create your own list, please read the [create list instructions](
 - [CLI Workshoppers/Adventures](https://github.com/therebelrobot/awesome-workshopper)
 - [Learn to Program](https://github.com/karlhorky/learn-to-program)
 - [Speaking](https://github.com/matteofigus/awesome-speaking)
+- [Tech Videos](https://github.com/lucasviola/awesome-videos)
```

Comment:
No, it's clearly limited to tech videos. I'm asking you to make the repo name and readme heading to reflect so too.